apis/stash/v1beta1/validator.go

```diff
@@ -67,3 +76,99 @@ func multipleRuleWithEmptyTargetHostError(ruleIndexes []int) string {
 	}
 	return fmt.Sprintf("%d rules found with empty targetHosts (Rules: %s)", len(ruleIndexes), ids)
 }
+
+func (b BackupBlueprint) IsValid() error {
+	// We must ensure the following:
+	// 1. Spec.schedule
+	// 2. Spec.Backend.StorageSecretName
+	// 3.
+	if b.Spec.Schedule == "" {
+		return fmt.Errorf("\n\t" +
+			"Error:  Invalid BackupBlueprint specification.\n\t" +
+			"Reason: BackupConfiguration Schedule is not specified\n\t" +
+			"Hints: Schedule is a cron expression like \"* * * * *\"")
```

Comment:
`"Hints: Provide a cron expression(i.e. \"* */5 * * *\") in 'spec.schedule' filed."`